olmo/config.py

```diff
@@ -243,6 +242,14 @@ class ModelConfig(BaseConfig):
     The number of self-attention heads.
     """
 
+    n_kv_heads: Optional[int] = None
+    """
+    The number of heads to use for keys and values.
```

Comment:
```suggestion
    The number of heads to use for keys and values. Defaults to `n_heads`.
```

Reply:
Done, I just can't click "commit" here for some reason.